store/storetest/post_store.go

```diff
@@ -94,6 +94,7 @@ func testPostStoreSave(t *testing.T, ss store.Store) {
 		replyPost.Message = "zz" + model.NewId() + "b"
 		replyPost.RootId = rootPost.Id
 
+		time.Sleep(2 * time.Millisecond)
```

Comment:
Leaving a comment with a link to the JIRA would help other people understand why this `sleep` was needed is not a hack.